node/src/runtime/testnet.rs

```diff
@@ -231,7 +231,32 @@ pub fn testnet_genesis(
 				session_time_in_seconds: 4 * 3600, // 4 hours
 			},
 		},
-		assets: Default::default(),
+		assets: AssetsConfig {
```

Comment:
Please, create migrations for this as well

Reply:
sure

Reply:
migrations is not work for assets module, but we can use sudo to init assets.